freqtrade/data/metrics.py

```diff
@@ -118,7 +118,7 @@ def _calc_drawdown_series(
 ) -> pd.DataFrame:
     max_drawdown_df = pd.DataFrame()
     max_drawdown_df["cumulative"] = profit_results[value_col].cumsum()
-    max_drawdown_df["high_value"] = max_drawdown_df["cumulative"].cummax()
+    max_drawdown_df["high_value"] = np.maximum(0, max_drawdown_df["cumulative"].cummax())
```

Comment:
are you sure about this?

this will limit `high_value` to be negative ... shouldn't this be `np.minimum()` - so we assume a high value of 0 or higher?